reactioncommerce/reaction-next-starterkit
# Add cart page and placeholder checkout page.

Resolves: #140 
Impact: minor

### Summary
Adds the shopping cart page and cart items related components including a list of items in the cart, a cart summary and checkout button.

**Testing**
1. Verify page is responsive across desktop and mobile devices
2. Verify styles and layout match designs.
